.github/workflows/osx_win_python_builds.yml

```diff
@@ -0,0 +1,369 @@
+name: Build OSX, Win & Python bindings
```

Comment:
I merged the workflow files for OSX & Win and added the linux python bindings job as well. That's mostly because of PyPI authentication, which is handled automatically with github, so no need to share passwords etc. Initially I thought it could only register a single workflow file, but turned out it could do more. Anyways, it's fine to group them into a single file, their `on:` is the same mostly

Reply:
I like the idea of running the macOS tests here, but I guess we might run the small risk of having out-of-sync wheels across platforms if those tests fail. Too bad we can't test windows :smile: 

Also, is there a way we can tell the exact commit the `-git` wheels were produced from, or does `__version__` just mimic what's in  `valhalla.h`?

Reply:
> That's not gonna happen, all jobs have to succeed for the wheels to be uploaded. 
Ah I didn't catch that, makes sense!